docs/taxonomy.md

```diff
@@ -24,8 +24,16 @@ Now, the tag is visible in the taxonomy tree.
 
 ### Assign tag to content from taxonomy tree
 
-When you edit or create a Content Type, in the section **Select tags**, you can add a tag by typing or clicking **Select from tree**.
-You can add many tags.
+Taxonomy is controlled and configured by the Field type **Taxonomy Entry Assignment** settings, so you can use multiple instances
+of the Field type with different taxonomies, for example, tags and product categories in the same Content Type. 
+To be able to assign tags to a Content, first, you need to add a Taxonomy Field type.
```

Comment:
Please refer to the fieldtype as `Taxonomy Entry Assignment Fieldtype` because there is another fieldtype called `Taxonomy Entry FieldType` which may be confusing to some.